app/containers/message/Attachments.tsx

```diff
@@ -58,6 +59,7 @@ const Attachments: React.FC<IMessageAttachments> = React.memo(
 	({ attachments, timeFormat, showAttachment, style, getCustomEmoji, isReply, author }: IMessageAttachments) => {
 		const { theme } = useTheme();
 		const { translateLanguage } = useContext(MessageContext);
+		const cdnPrefix = useAppSelector(state => state.settings.CDN_PREFIX as string);
```

Comment:
In the three different components you use cdnPrefix as a prop, and you only use this cdnPrefix in one function.
The best thing would be to do the same as we do in other functions and use the auxStore to access the redux of that function.
cdnPrefix is not something that will change, so it is not really necessary to listen to redux changes.
In addition to making your code MUCH cleaner, since you changed 3 different files using the same property.

Reply:
In audio, put the redux in the component and that's it.